start

```diff
@@ -108,6 +108,7 @@ function main() {
         echo "Installed horizon ..."
         if [ ! -z "$SOROBAN_RPC_SOURCE_VOLUME" ]; then
             pushd "$SOROBAN_RPC_SOURCE_VOLUME"
+            go generate ./cmd/soroban-rpc/...
```

Comment:
can we invoke the `build-soroban-rpc` target from soroban-tools Makefile here instead, that target would encapsulate latest config/steps needed for build rather than replicating it here also?

Reply:
Great idea! I didn't realize there was a makefile